lib/foreman/renderer.rb

```diff
@@ -26,6 +26,18 @@ def template_logger
       @template_logger ||= Foreman::Logging.logger('templates')
     end
 
+    def host_info(*path)
+      @info ||= @host.info
+      info = @info
+      path.each { |step| info = info.fetch step }
+      info
+    end
+
+    def host_param(*path)
```

Comment:
OK, I agree it's preferable over using `@host`, but the change needs to:

1. Also include replacements for `param_true?` and `param_false?`, since these are used on `@host`.
1. Preferably include a deprecation warning for the old methods when called from templates (perhaps set a thread variable during rendering, then it's easy to only deprecate when called that way) so we can indicate which method is preferred.